src/System.Private.CoreLib/shared/System/Range.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public readonly struct Range
+    {
+        public Index Start { get; }
+        public Index End { get; }
+
+        private Range(Index start, Index end)
+        {
+            this.Start = start;
```

Comment:
The CoreCLR/CoreFX convention is not use explicit `this`.

Reply:
Done.